ApiBundle/Controller/ContentController.php

```diff
@@ -32,17 +33,44 @@ class ContentController extends BaseController
      * @Api\Serialize()
      *
      * @return FacadeInterface
+     * @throws ContentNotFoundHttpException
      */
     public function showAction(Request $request, $contentId)
     {
+        $content = $this->findOneContent($contentId, $request->get('language'), $request->get('version'));
+
+        if (!$content) {
+            throw new ContentNotFoundHttpException();
+        }
+
+        return $this->get('open_orchestra_api.transformer_manager')->get('content')->transform($content);
+    }
+
+    /**
+     * @param Request $request
+     * @param string  $contentId
+     *
+     * @Config\Route("/{contentId}/or/create", name="open_orchestra_api_content_show_or_create")
```

Comment:
I find this route strange, I find something like /show-or-create/{contentId} more conventional